src/services/evaluation-administration-service.ts

```diff
@@ -234,7 +234,84 @@ export const close = async (id: number) => {
   )
 }
 
-export const sendReminder = async (id: number) => {
+export const sendReminderByEvaluator = async (id: number, evaluator_id: number, email: string) => {
+  const evaluationAdministration = await EvaluationAdministrationRepository.getById(id)
+
+  if (evaluationAdministration === null) {
+    throw new CustomError("Id not found", 400)
+  }
+
+  const emailTemplate = await EmailTemplateRepository.getByTemplateType(
+    "Performance Evaluation Reminder"
+  )
+
+  if (emailTemplate === null) {
+    throw new CustomError("Template not found", 400)
+  }
+
+  const scheduleEndDate = format(
+    evaluationAdministration.eval_schedule_end_date ?? new Date(),
+    "EEEE, MMMM d, yyyy"
+  )
+
+  const externalEvaluator = await ExternalUserRepository.getByFilters({
+    id: evaluator_id,
+    email,
+  })
+
+  const isExternal: boolean = externalEvaluator !== null
+
+  const filter = {
+    evaluation_administration_id: evaluationAdministration.id,
+    for_evaluation: true,
+    status: {
+      in: [EvaluationStatus.Open, EvaluationStatus.Ongoing],
+    },
```

Comment:
and just add `...(is_external ? { external_evaluator_id: user_id } : { evaluator_id: user_id })` here